### PR TITLE
Fix android scrolling bug

### PR DESCRIPTION
**Pull request by thenaterhood**
This needs to be vetted on iOS before being merged. It works on Android.

And no, I'm not sure why it wants it to be MyApp.* in the requires but it wouldn't build for me using DevCycleMobile.*.

See:

http://www.ladysign-apps.com/developer/chrome-43-breaks-sencha-touch-modern-toolkit-and-how-to-fix-it/
http://www.joshmorony.com/the-chrome-43-update-broke-sencha-touch-apps-heres-how-to-fix-it/

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -37,7 +37,9 @@ Ext.application({
         'DevCycleMobile.store.Misc',
         'DevCycleMobile.store.Credits',
         'DevCycleMobile.store.AboutTracking',
-		'DevCycleMobile.store.MyGroups'
+	'DevCycleMobile.store.MyGroups',
```
Comment (bbesmanoff) [inline app.js]: The spacing is off here

**Comment by bbesmanoff** [comment]
When I build this branch, there are autogenerated changes in `app.js`

**Comment by bbesmanoff** [comment]
Seems to work in the iPhone 5 simulator

Approved :+1:

**Comment by thenaterhood** [comment]
Whitespace is fixed.

What changed in app.js? When I do a build nothing in it changes.

**Comment by bbesmanoff** [comment]
Turns out it was in `bootstrap.js`.  Patch attached

[out.txt](https://github.com/TourTrak/devcycle-mobile-app/files/200501/out.txt)

**Comment by thenaterhood** [comment]
Oh gotcha, okay. That likely should get committed as well.

File: app/util/SizeMonitor.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        } else if (Ext.browser.is.IE11) {
+            return new namespace.Scroll(config);
+        } else {
+            return new namespace.Scroll(config);
+        }
+    }
+}, function () {
+    // 
+    console.info("Ext.util.SizeMonitor temp. fix is active");
+    //
```
Comment (CShapleigh) [inline app/util/SizeMonitor.js]: whats with the '//'s

Comment (thenaterhood) [inline app/util/SizeMonitor.js]: Copypasta from the solution posted by sencha's devs. I can remove them.

Comment (CShapleigh) [inline app/util/SizeMonitor.js]: If you make another change, include removing them in it. Otherwise I wouldn't worry about it

**Comment by CShapleigh** [comment]
Tested, confirmed, approved. :+1:

**Pull request merged**